### PR TITLE
Add manifest for Hypatia (com.github.HypatiaProject.hypatia)

### PR DESCRIPTION
**Pull request by nathandyer**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.HyptiaProject.hypatia.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "hypatia",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/HypatiaProject/hypatia.git"
```
Comment (hfiguiere) [inline com.github.HyptiaProject.hypatia.json:34]: you need a specific revision or tag.

File: com.github.HyptiaProject.hypatia.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (hfiguiere) [inline com.github.HyptiaProject.hypatia.json:41]:
```suggestion
    ]
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.HypatiaProject.hypatia

**Comment by hfiguiere** [comment]
The manifest is not properly named. It has a typo. That's why the build is failing.

**Comment by hfiguiere** [comment]
bot, build com.github.HypatiaProject.hypatia

**Comment by hfiguiere** [comment]
appdata fail to validate

```
builddir/files/share/appdata/com.github.HypatiaProject.hypatia.appdata.xml: FAILED:
• tag-missing           : <url> is not present
Validation of files failed
```

**Comment by nathandyer** [comment]
My apologies, I validated it with appstream-util before pushing the release, but apparently I was using the relaxed option. Ran again without the relaxed option, found the issue, and corrected it.

Should be good to try building again.

**Comment by hfiguiere** [comment]
bot, build com.github.HypatiaProject.hypatia

**Comment by nathandyer** [comment]
Tested the build, everything looks good to me!

**Pull request closed without merging**